packages/editor/src/lib/hooks/useTLStore.ts

```diff
@@ -24,7 +24,7 @@ function createStore(opts: UseTLStoreOptions) {
 
 /** @public */
 export function useTLStore(
-	opts: TLStoreOptions & { snapshot?: TLEditorSnapshot | TLStoreSnapshot }
+	opts: TLStoreOptions & { snapshot?: Partial<TLEditorSnapshot> | TLStoreSnapshot }
```

Comment:
Here we could even use `UseTLStoreOptions`, but maybe we want to expose the exploded type for better DX?

Reply:
Would make a difference in this case? I'm fine either way.

Reply:
No, not really 🤷‍♂️